manifest.php

```diff
@@ -26,7 +26,7 @@
     'label' => 'QTI item model',
 	'description' => 'TAO QTI item model',
     'license' => 'GPL-2.0',
-    'version' => '2.7.6',
+    'version' => '2.7.7',
```

Comment:
In develop, the current version is already 2.7.7..
This has been updated by the PR : https://github.com/oat-sa/extension-tao-itemqti/pull/285
Should you not update the version to 2.7.8 or more ?